packages/web-react/src/components/Dropdown/useDropdown.ts

```diff
@@ -0,0 +1,49 @@
+import { useState, MutableRefObject } from 'react';
+import { ClickEvent } from '../../types';
+import { useClickOutside } from '../../hooks';
+
+export interface UseDropdownProps {
+  /** dropdown element reference */
```

Comment:
I think this is enough self-documenting code.
Is there really need to have these following comments that explains obvious?

Reply:
I was just doing what was on other components like `useButtonStyleProps`, `useButtonAriaProps`...